omf-python/src/lib.rs

```diff
@@ -23,11 +23,13 @@ fn omf_python(m: &Bound<'_, PyModule>) -> PyResult<()> {
     m.add_class::<PyAttributeDataCategory>()?;
     m.add_class::<PyArrayIndex>()?;
```

Comment:
Maybe you don't have tot change this here, but I think these objects would be clear about what they are doing if the name ends in array i.e. `PyIndexArray` rather than `PyArrayIndex`, likewise `PyTriangleArray`.  @antbrown

Reply:
Yep, once merged I can update the names of these things, sounds good to me :+1: